theme/templates/page.html

```diff
@@ -3,5 +3,15 @@
 {% block title %}{{ page.title }}{% endblock title %}
 
 {% block content %}
-    {{ page.content }}
+	{% if page.body_class %}
+		<!-- do nothing -->
+	{% else %}
+		<section class="col-12 secondary community">
```

Comment:
This is better, but you are still missing something. The rendered markdown page looks different from the one currently online. 

Also convert tabs to spaces.